Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ListViewExtensions/ListViewExtensionsPage.xaml.cs

```diff
@@ -10,19 +12,61 @@ namespace Microsoft.Toolkit.Uwp.SampleApp.SamplePages
 {
     public sealed partial class ListViewExtensionsPage : Page, IXamlRenderListener
     {
+        private ListView sampleListView;
+        private TextBlock indexInput;
+        private TextBlock itemPlacementInput;
+        private TextBlock disableAnimationInput;
+        private TextBlock scrollIfVisibileInput;
+        private TextBlock additionalHorizontalOffsetInput;
+        private TextBlock additionalVerticalOffsetInput;
+
         public ListViewExtensionsPage()
         {
             this.InitializeComponent();
+            Load();
         }
 
-        public async void OnXamlRendered(FrameworkElement control)
+        public void OnXamlRendered(FrameworkElement control)
         {
-            var sampleListView = control.FindChildByName("SampleListView") as ListView;
+            sampleListView = control.FindChildByName("SampleListView") as ListView;
+            indexInput = control.FindChildByName("IndexInput") as TextBlock;
+            itemPlacementInput = control.FindChildByName("ItemPlacementInput") as TextBlock;
+            disableAnimationInput = control.FindChildByName("DisableAnimationInput") as TextBlock;
+            scrollIfVisibileInput = control.FindChildByName("ScrollIfVisibileInput") as TextBlock;
+            additionalHorizontalOffsetInput = control.FindChildByName("AdditionalHorizontalOffsetInput") as TextBlock;
+            additionalVerticalOffsetInput = control.FindChildByName("AdditionalVerticalOffsetInput") as TextBlock;
 
             if (sampleListView != null)
             {
-                sampleListView.ItemsSource = await new Data.PhotosDataSource().GetItemsAsync();
+                sampleListView.ItemsSource = GetOddEvenSource(201);
+            }
+        }
+
+        private void Load()
+        {
+            SampleController.Current.RegisterNewCommand("Start Smooth Scroll", (sender, args) =>
+            {
+                var index = int.Parse(indexInput.Text);
+                var itemPlacement = (ItemPlacement)Enum.Parse(typeof(ItemPlacement), itemPlacementInput.Text);
```

Comment:
Input can't be invalid because it is bind from Property Panel. But anyway I changed to `ComboBox` to avoid parsing.

https://github.com/windows-toolkit/WindowsCommunityToolkit/blob/b57e6df24ca205561d12e815552496a60378db61/Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ListViewExtensions/ListViewExtensionsXaml.bind#L22-L23